docs/support-policy.md

```diff
@@ -6,7 +6,7 @@
 # Packages support policy
 
 `tezos-packaging` provides packages for Ubuntu, Fedora and macOS for each stable and release-candidate
-version of Octez.
+version of Octez. `Ubuntu` packages can also be used on WSL, but their support is limited. Functionality limitations are described in the corresponding sections.
```

Comment:
:thinking: I'm not sure that we should mention this here.

Overall I think that we should treat Windows support like we do Raspberry Pi OS, we don't offer specific native packages for them, but Ubuntu ones can be used (with some limitations).

So, I suggest reverting this change and instead adding:
- a "Ubuntu packages on WSL" section in the [Ubuntu installation docs](https://github.com/serokell/tezos-packaging/blob/master/docs/distros/ubuntu.md)
- a "Windows using WSL" link to that section in the [Installing tezos](https://github.com/serokell/tezos-packaging#installing-tezos)

As for what to put there, well, I think it should point at the [Ubuntu WSL](https://ubuntu.com/wsl) docs and then list how to perform the installation with that being set up (once again, much like we do for Raspberry Pi OS).